src/api/controllers/articlesController.js

```diff
@@ -92,7 +92,7 @@ class articlesController {
     const allArticle = await articles.getAllArticle();
 
     if (!allArticle[0]) {
-      return res.status(404).send({ error: 'Whoops! No Articles found!' });
+      return res.status(404).send({ message: 'Whoops! No Articles found!' });
```

Comment:
I prefer `error` as the key to displaying errors.

Reply:
@Quantum-35      ==> the response returns an empty array. can i change the status code from 404 to 200 with a message instead of error? could I have your attention on this, please?